src/DocScan/DocScanClient.php

```diff
@@ -129,13 +129,13 @@ public function deleteMediaContent(string $sessionId, string $mediaId): void
     }
 
     /**
-     * Gets a list of supported documents.
```

Comment:
Ah, presume this comment wasn't intentionally removed?

Reply:
I see, I will add it. Thx.